linera-chain/src/data_types.rs

```diff
@@ -364,6 +365,27 @@ impl OutgoingMessage {
     }
 }
 
+/// The execution result of a single operation.
+#[derive(Debug, Default, PartialEq, Eq, Hash, Clone, Serialize, Deserialize)]
+pub struct OperationResult(
+    #[debug(with = "hex_debug")]
+    #[serde(with = "serde_bytes")]
+    Vec<u8>,
+);
+
+impl From<Vec<u8>> for OperationResult {
+    fn from(operation_outcome: Vec<u8>) -> Self {
+        OperationResult(operation_outcome)
+    }
+}
+
+impl<'de> BcsHashable<'de> for OperationResult {}
+
+doc_scalar!(
```

Comment:
I agree with @ma2bd here:

```suggestion
bcs_scalar!(
```

Reply:
Why was this resolved?